src/towncrier/build.py

```diff
@@ -141,30 +141,49 @@ def __main(
         top_line = config["title_format"].format(
             name=project_name, version=project_version, project_date=project_date
         )
+        render_title_with_fragments = False
+        render_title_separately = True
+    elif config["title_format"] == False:  # noqa: E712
```

Comment:
Use the identity check for booleans and None:
```suggestion
    elif config["title_format"] is False:
```